src/components/Swiper/Swiper.tsx

```diff
@@ -324,14 +325,22 @@ export const Swiper = (inProps: SwiperProps) => {
   const setActiveSlideByStep = useCallback(
     (step: number, options?: { smooth?: boolean }) => {
       if (container.current) {
+        if (active === to && step === 1 && loop) {
```

Comment:
Я бы немного перестраховался, тут может быть число больше 1 =)
```suggestion
        if (active === to && step > 0 && loop) {
```